waku-sys/build.rs

```diff
@@ -36,6 +36,7 @@ fn build_go_waku_lib(go_bin: &str, project_dir: &Path) {
     set_current_dir(vendor_path).expect("Moving to vendor dir");
     Command::new(go_bin)
         .env("CGO_ENABLED", "1")
+        .env("GOCACHE", "off")
```

Comment:
can we add it just when we detect building in docs.rs? there is some flag they provide:

https://docs.rs/about/builds#detecting-docsrs

Reply:
Updated with the suggestion.